README.md

```diff
@@ -1 +1,27 @@
-# hw01
\ No newline at end of file
+# hw01 This is a line from RStudio
+
+##A Brief Biography of Myself, Wael Taji Miller
+
+###The Early Days
+
+In the beginning, Little Wael grew up in a dirty and poor part of England called West London. In his early years, he demonstrated his potential through a number of impressive achievements, which include:
+
+- Starting fights
+- Rarely washing
+- Being profoundly disagreeable
+- Failing school
+- Getting back into school
+- Getting really good grades and stuff
+
+Because of his impressive career background and talented skill portfolio, Little Wael was offered a place at the University of Cambridge, which is way older than your whole entire country and stuff (see Figure 1 for details).
+
+![Figure 1: How Cambridge University is Literally Even Older Than Your Whole Entire Country and Stuff](https://raw.githubusercontent.com/waeltmiller/hw01-1/master/Screen%20Shot%202019-10-04%20at%2018.17.24.png)
+
+###Getting Into Science
+Little Wael soon evolved into his final form as Bigger Wael (though alas, he would never grow taller than [five feet and eleven inches in height](https://i.4pcdn.org/pol/1510707466870.jpg)). Although Bigger Wael acheived top grades early on in his humanities degree,[^1] he ultimately came to the conclusion that any debate on real-world issues that eschews evidence over anecdote and is unhindered by the burden of proof is much akin to playing World of Warcraft: there seems to be a lot of action going on, but the ~~lvlcap~~ goalposts keep expanding and you never really get anywhere. 
```

Comment:
Unfortunately, the github flavor of markdown doesn't support footnoting in this style. You're correct that normally [^x] marks the formation of a new footnote, but if you notice on the rendered readme page, the footnote doesn't link down to the bottom of the page as you'd expect it would!